src/Layout/Presentation/DistrictOverview/DistrictOverview.tsx

```diff
@@ -30,7 +32,8 @@ export class DistrictOverview extends React.Component<DistrictOverviewProps, {}>
                 {data.find((entry) => entry.votesPerSeat === lowestVotingPower)!.name}
             </span>
         );
-        const mostVulnerable = getMostVulnerableSeatByQuotient(data);
+        const calculateVulnerable = isQuotientAlgorithm(this.props.algorithm);
+        const mostVulnerable = calculateVulnerable ? getMostVulnerableSeatByQuotient(data) : undefined;
```

Comment:
Why does that return "undefined"? That seems suspicious.

Reply:
We return undefined when the algorithm is not a quotient based algorithm, as the closest seat does not always make sense for largest fraction. For example, all seats may be won in a single round, there is no definition of which seat was last.